docs/UserGuide.md

```diff
@@ -24,30 +25,32 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
    
-   * **`list`** : Lists all FlashCards.
-   
-   * **`open`**`1` : Opens the 1st question shown in the current list.
+   * **`add`**`q/Plants give out ___ when they photosynthesise? ans/Oxygen t/Biology` :  Adds an open ended question `Plants give out ___ when they photosynthesise?` with answer `Oxygen` and tagged to `Biology`. 
```

Comment:
Could you change it from

> Adds an open ended question `Plants give out ___ when they photosynthesise?` with answer `Oxygen` and tagged to `Biology`.

to "Adds a flashcard with an open ended question `Plants give out ___ when they photosynthesise?`. The flashcard will have an answer  `Oxygen` and has a tag `Biology`."